<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>300</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nunito Sans Normal ExtraBold</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>500</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nunito Sans Normal SemiBold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 10px;</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>500</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nunito Sans Normal SemiBold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 10px;</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>350</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nunito Sans Normal ExtraBold</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 10px;
	border: 1px solid rgb(21, 8, 204);
	background-color: rgb(21, 8, 204);
	color: white;
}

QPushButton:hover{
	color: rgb(21, 8, 204);
	background-color: white;
}

QPushButton:pressed{
	padding-top: 3px;
	padding-left: 3px;
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
